jdaviz/configs/imviz/plugins/rotate_canvas/rotate_canvas.py

```diff
@@ -0,0 +1,107 @@
+from traitlets import Bool, observe
+
+from jdaviz.configs.imviz.wcs_utils import get_compass_info
+from jdaviz.core.custom_traitlets import FloatHandleEmpty
+from jdaviz.core.events import AddDataMessage, RemoveDataMessage, CanvasRotationChangedMessage
+from jdaviz.core.registries import tray_registry
+from jdaviz.core.template_mixin import PluginTemplateMixin, ViewerSelectMixin
+from jdaviz.core.user_api import PluginUserApi
+
+__all__ = ['RotateCanvas']
+
+
+@tray_registry('imviz-rotate-canvas', label="Canvas Rotation", viewer_requirements='image')

```

Comment:
`viewer_requirements='image'` eh? Are you sure users won't want to rotate spectrum viewer in the future so they don't have to subtract a monotonously increasing/decreasing continuum? 😉